src/main/webapp/synchronizer.js

```diff
@@ -19,26 +19,28 @@ let lastTime;
 
 const TIME_INTERVAL_MS = 100;
 
-/**
- * Starts timer which broadcasts current video time every
- * `TIME_INTERVAL` milliseconds.
- */
-export function startVideoSyncTimer() {
-  window.setInterval(() => {
-    sync(window.videoPlayer.getCurrentTime());
-  }, /* ms= */ TIME_INTERVAL_MS);
-}
+export default class Synchronizer {
```

Comment:
Remember that classes should have JsDocs

https://google.github.io/styleguide/jsguide.html#jsdoc-class-comments